scio-elasticsearch/src/main/scala/com/spotify/scio/elasticsearch/package.scala

```diff
@@ -56,17 +56,22 @@ package object elasticsearch {
      */
     def saveAsElasticsearch(esOptions: ElasticsearchOptions,
                             f: T => IndexRequest):Future[Tap[T]] = {
-      val numOfWorkers = self.context.pipeline.getRunner match {
-        case _: DirectRunner => 1
-        case _: DataflowRunner =>
+      val runner = self.context.options.getRunner
+      val numOfWorkers = {
+        if (classOf[DirectRunner] isAssignableFrom(runner)) {
```

Comment:
`isAssignableFrom runner`? Infix should be `a op b` and postfix `a.op(b)`. Don't mix them.